### DISCUSSION TITLE
Difference between two similar dependabot settings

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#61966] [Code Security] [answered]
### Select Topic Area

Question

### Body

Does anyone know what is the difference between the following two settings?

![Screenshot 2023-07-27 at 6 01 39 PM](https://github.com/community/community/assets/50835081/d025302d-a88d-4738-8681-a6cbe3617acd)

**Comment by carogalvin** [comment, marked as answer]
@pantelis-karamolegkos Dependabot security updates are where Dependabot will detect an advisory associated with a version of a package you are using, send you an alert, and automatically open a PR to bump the package to the minimum version where the vulnerability is patched. Version updates will look at all your dependencies on a schedule you configure and try to bump all of them to latest. You can learn more in our documentation here: https://docs.github.com/en/code-security/dependabot
